css/index.css

```diff
@@ -65,143 +65,236 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
-
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
-
-* {
-    box-sizing: border-box;
-}
+/* reset end */
 
 html {
     font-size: 62.5%;
 }
 
 .container {
-	 display: flex;
-	 max-width: 880px;
-	 margin: 0 auto;
-	 flex-wrap: wrap;
-  } 
-  
-		
+    max-width: 880px;
+    margin: auto;
+}
+
+/* Nav Styles */
+
+
 
  nav {
-	display: flex;
-	min-width: 600px;
-	max-width: 1100px;
-	justify-content: space-between;
-    align-items: center;
- }
+    display: flex;
+    margin-top: 40px; 
+    justify-content: space-between;
+    align-items: baseline;
+
+}
 
+ .links {
+    display: flex;
+    justify-content: space-between;
+    color: #b0b0b0;
+    /* border: 2px solid red; */
+}
+
+.links a {
+    padding-right: 4rem;
```

Comment:
this is always going to create that horizontal scroll bar, when you do a padding, why dont you try making the parent container flex and doing space between or around, whatever you see fit